src/main/java/project/controllers/UserController.java

```diff
@@ -0,0 +1,47 @@
+package project.controllers;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import project.UserResponseDto;
+import project.model.User;
+import project.service.UserService;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+    @Autowired
+    UserService userService;
+
+    @RequestMapping("/inject")
+    public String inject() {
+        userService.add(new User("Bob", "1234"));
+        userService.add(new User("Alisa", "1234"));
+        userService.add(new User("Bogdan", "1234"));
+        userService.add(new User("Vasa", "1234"));
+        return "Users was injected";
+    }
+
+    @RequestMapping("/get/{userId}")
+    public UserResponseDto get(@PathVariable Long userId) {
+        return getUserDto(userService.getById(userId));
+    }
+
+    @RequestMapping("/get/")
```

Comment:
```suggestion
    @RequestMapping("/")
```
see REST API naming conventions